packages/ui-library/src/components/toggle-switch/index.stories.ts

```diff
@@ -413,7 +413,7 @@ Readonly.story = { name: ' ' };
 /**
  * ## Dependencies
  * ### Form Caption
- * The Toggle Switch component can display an optional hint message with or without icons. For more information have a look at the [Form Caption](/docs/design-system-web-components-internal-components-formcaptiongroup-formcaption--docs) component.
+ * The Toggle Switch component can display an optional hint message with or without icons. For more information have a look at the [Form Caption](/docs/components-form-caption-group-formcaption--docs) component.
```

Comment:
Link to the Form Caption is not working (it should not link to form caption group)

Reply:
done